README.md

```
@@ -24,8 +39,8 @@ See는 사용자들이 다양한 콘텐츠를 작성하고 공유하며 활발
    Spring Boot Docker Compose 지원으로 MySQL 컨테이너가 자동으로 실행됩니다.
 
 3. **애플리케이션 접속**
-   - API: http://localhost:8080
-   - Swagger UI: http://localhost:8080/swagger-ui.html
+   - API 서버: http://localhost:8080
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)